crates/runtimes/src/errors.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+pub type Result<T> = std::result::Result<T, RuntimeError>;
+
+#[derive(Debug)]
+pub enum RuntimeError {
+    ErrorCreatingWasiContext,
+    CannotReadModule,
+    ErrorReadingLocalModule,
+    FileHasInvalidExtension { extension: Option<String> },
+    InvalidWrapper,
+    IOError(std::io::Error),
+    MissingRuntimeForExtension { extension: String },
+    StoreError(wws_store::errors::StoreError),
+    WasiError(Option<wasmtime_wasi::Error>),
+}
```

Comment:
For consistency, I would rename some of these errors:

```suggestion
pub enum RuntimeError {
    WasiContextError,
    CannotReadModule,
    InvalidExtension { extension: Option<String> },
    InvalidWrapper,
    IOError(std::io::Error),
    MissingRuntime { extension: String },
    StoreError(wws_store::errors::StoreError),
    WasiError(Option<wasmtime_wasi::Error>),
}
```